logstash-core/src/main/java/org/logstash/common/Util.java

```diff
@@ -35,4 +47,38 @@ public static String bytesToHexString(byte[] bytes) {
 
         return hexString.toString();
     }
+
+    /**
+     * Unzips a ZipInputStream to a given directory
+     * @param input the ZipInputStream
+     * @param output path to the output
+     * @param omitParentDir omit the parent dir the zip is packaged with
+     * @throws IOException
+     */
+    public static void unzipToDirectory(final ZipInputStream input, final Path output, boolean omitParentDir) throws IOException {
```

Comment:
Minor -- I prefer putting code that's used only in tests somewhere in the test source tree.

Reply:
+1

Reply:
+1